knowledge-base/fileselect-disable-onselect.md

```diff
@@ -0,0 +1,72 @@
+---
+title: How to change the FileSelect button's caption and disable it during selection
+description: Hot to change FileSelect button's caption, while selecting and uploading. How to disable it during selection.
+type: how-to
+page_title: How to disable the FileSelect, while selecting and changing its caption's button.
+slug: fileselect-disable-onselect
+position:
+tags: fileselect, upload, select, disable, loader
+ticketid: 1639447
+res_type: kb
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>
+                FileSelect for Blazor, <br />
+                Loader for Blazor <br />
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+
+## Description
+
+This KB article answers the following questions:
+
+* How to change the FileSelect's button caption, when the user select or upload multiple files?
+* How to disable the FileSelect during upload of multiple files?
```

Comment:
Is this about disabling the entire component or an element/feature of it?

Reply:
It disables the entire component.

Reply:
Then it might be best to say "How to disable FileSelect during..." or "How to disable the FileSelect component during..."